src/Funds/FundDetails/FundDetailsContainer.js

```diff
@@ -143,24 +159,49 @@ const FundDetailsContainer = ({
     fetchFund();
   };
 
+  const goToTransactions = useCallback(
+    () => {
+      history.push(`${FUNDS_ROUTE}/view/${params.id}/budget/${currentBudget.id}${BUDGET_TRANSACTIONS_ROUTE}`);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [params.id, currentBudget],
+  );
+
   const toggleTagsPane = () => setIsTagsPaneOpened(!isTagsPaneOpened);
 
   const renderActionMenu = useCallback(
     ({ onToggle }) => (
       <MenuSection id="fund-details-actions">
         <DetailsEditAction
           perm="finance.funds.item.put"
-          onEdit={editGroup}
+          onEdit={editFund}
           toggleActionMenu={onToggle}
         />
+        {currentBudget?.id && (
+          <Button
+            buttonStyle="dropdownItem"
+            data-test-details-view-transactions-action
+            onClick={() => {
+              goToTransactions();
+              onToggle();
+            }}
+          >
+            <Icon
+              size="small"
+              icon="eye-open"
```

Comment:
is there any difference with `IconButton` usage? https://ux.folio.org/storybook/?selectedKind=IconButton&selectedStory=Basic%20Usage&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

Reply:
`IconButton` - just icon. But here we need icon + label